src/controller/python/test_client/rpc_client.py

```diff
@@ -109,6 +109,53 @@ async def main():
         fabric_id = server_1.get_fabric_id()
         print(f"fabric_id: {fabric_id}")
 
+        # Network Provisioning commands
```

Comment:
We should start making these RPC commands into methods so we can easily comment/uncomment these